app/grandchallenge/algorithms/forms.py

```diff
@@ -40,6 +44,10 @@
 from grandchallenge.subdomains.utils import reverse_lazy
 
 
+class ModelFactsTextField(Field):
+    template = "algorithms/model_facts_field.html"
```

Comment:
The help texts are bit longer for the model facts fields and we did not want them to get lost underneath the markdown helptext that is printed for each field already. Hence the custom template.

Reply:
You can create a docstring for this new class to let future developers know this :-)